doc/project/dev_env_and_tools.rst

```diff
@@ -263,7 +263,7 @@ These are the labels we currently have, grouped by applicability:
 Labels applicable to issues only
 ================================
 
-* ``priority: {high|medium|low}``
+* ``priority:{high|medium|low}``
```

Comment:
I think this is not correct. I see a space on GitHub. Here is an example search: https://github.com/zephyrproject-rtos/zephyr/issues?q=is%3Aissue+is%3Aopen+label%3A%22priority%3A+high%22

Reply:
Fixed in https://github.com/zephyrproject-rtos/zephyr/pull/44774/commits/cab8ea2758bb3f11808fc10b046d7cfa5d797508 .